queries.sql

```diff
@@ -1,195 +1,6 @@
-FROM animals A
-JOIN owners O
-ON A.owner_id = O.id
-GROUP BY O.full_name
-ORDER BY COUNT(A.id) DESC
-LIMIT 1;
-
--- Write queries to answer the following:
-
--- Who was the last animal seen by William Tatcher?
-SELECT vets.name, animals.name, visits.visit_date
-FROM visits
-JOIN animals
-ON visits.animals_id = animals.id
-JOIN vets
-ON visits.vets_id = vets.id AND vets.name = 'William Tatcher'
-ORDER BY visits.visit_date DESC
-LIMIT 1;
-
--- How many different animals did Stephanie Mendez see?
-SELECT COUNT(animals.id)
-FROM visits
-JOIN animals
-ON visits.animals_id = animals.id
-JOIN vets
-ON visits.vets_id = vets.id AND vets.name = 'Stephanie Mendez';
-
--- List all vets and their specialties, including vets with no specialties.
-SELECT vets.name, species.name
-FROM vets
-LEFT JOIN specializations
-ON specializations.vets_id = vets.id
-LEFT JOIN species
-ON specializations.species_id = species.id;
-
--- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
-SELECT vets.name, animals.name, visits.visit_date
-FROM visits
-JOIN animals
-ON visits.animals_id = animals.id
-JOIN vets
-ON visits.vets_id = vets.id AND vets.name = 'Stephanie Mendez'
-WHERE visits.visit_date BETWEEN '2020-04-01' AND '2020-08-30';
-
--- What animal has the most visits to vets?
-SELECT animals.name, COUNT(id)
-FROM visits
-JOIN animals
-ON visits.animals_id = animals.id
-GROUP BY animals.name
-ORDER BY COUNT(id) DESC
-LIMIT 1;
-
--- Who was Maisy Smith's first visit?
-SELECT vets.name, animals.name, visits.visit_date
-FROM visits
-JOIN animals
-ON visits.animals_id = animals.id
-JOIN vets
-ON visits.vets_id = vets.id AND vets.name = 'Maisy Smith'
-ORDER BY visits.visit_date ASC
-LIMIT 1;
-
--- Details for most recent visit: animal information, vet information, and date of visit.
-SELECT visits.visit_date, animals.name, vets.name
-FROM visits
-JOIN animals
-ON visits.animals_id = animals.id
-JOIN vets
-ON visits.vets_id = vets.id
-ORDER BY visits.visit_date DESC
-LIMIT 1;
-
--- How many visits were with a vet that did not specialize in that animal's species?
-SELECT COUNT(*)
-FROM visits
-JOIN animals
-ON visits.animals_id = animals.id
-JOIN species
-ON animals.species_id = species.id
-JOIN vets
-ON visits.vets_id = vets.id
-LEFT JOIN specializations
-ON specializations.vets_id = vets.id
-WHERE specializations.species_id != animals.species_id OR specializations.species_id IS NULL;
-
-
--- What specialty should Maisy Smith consider getting? Look for the species she gets the most.
-SELECT species.name, COUNT(species.name)
-FROM visits
-JOIN vets
-ON visits.vets_id = vets.id AND vets.name = 'Maisy Smith'
-JOIN animals
-ON visits.animals_id = animals.id
-JOIN species
-ON animals.species_id = species.id
-GROUP BY(species.name)
-ORDER BY COUNT(species.name) DESC
-LIMIT 1;
+SELECT COUNT(*) FROM visits where animal_id = 4;
+SELECT * FROM visits where vet_id = 2;
+SELECT * FROM owners where email = 'owner_18327@mail.com';
```

Comment:
- [ ] You should Write queries (using JOIN) to answer the following questions 👇🏻  Here are the requirements:

![image](https://github.com/elad237/Vet-clinic-database/assets/79058364/ada79821-959e-4635-8c5c-c1d1d750fb91)

- Please adress this. 👍🏻